src/contrib/cluster/Akka.DistributedData/Replicator.cs

```diff
@@ -601,7 +613,7 @@ private void ReceiveGet(IKey key, IReadConsistency consistency, object req)
                 else Sender.Tell(new GetSuccess(key, req, localValue.Data));
             }
             else
-                Context.ActorOf(ReadAggregator.Props(key, consistency, req, _nodes, _unreachable, localValue, Sender)
+                Context.ActorOf(ReadAggregator.Props(key, consistency, req, NodesForReadWrite(), _unreachable, !_settings.PreferOldest, localValue, Sender)
```

Comment:
So on each read aggregation we might run that query - ok, don't think that's all that frequent for the most part

Reply:
Shuffle is enabled when `PreferOldest == false` ?

Reply:
Exactly. When `PreferOldest == true`, nodes are sorted by oldest.
When `PreferOldest == false`, nodes are taken in a random order